src/__tests__/__snapshots__/index.js.snap

```diff
@@ -116,7 +122,7 @@ exports[`merges composed component styles for reasonable overrides 1`] = `
 }
 
 <div
-  class="other classes are not removed css-1k8eh41"
+  class="other classes are not removed css-1k8eh41 glamorous_glamorous_glamorous_div__"
```

Comment:
Instead of  `glamorous_glamorous_glamorous_div__` we could display `3*glamorous_div__`